examples/platforms/da15000/radio.c

```diff
@@ -95,15 +97,17 @@ void da15000RadioInit(void)
 void otPlatRadioGetIeeeEui64(otInstance *aInstance, uint8_t *aIeeeEui64)
 {
     (void)aInstance;
+    uint32_t *factoryTestTimeStamp  = (uint32_t *) FACTORY_TEST_TIMESTAMP;
+    uint32_t *factoryTestId         = factoryTestTimeStamp + 1;
```

Comment:
Using ```#define FACTORY_TEST_ID  0x7F8EA0C``` would be more explicit, and make it clearer where this value is coming from.  

Incrementing `factoryTestTimeStamp` only makes sense if you have access to the data sheet and know that the factory test ID is the next word.  Since this is intended as an example, it would be nice to be as explicit as possible to make it clear to readers how the example works.